cms/static/js/views/video/translations_editor.js

```diff
@@ -89,7 +89,7 @@ function($, _, AbstractEditor, FileUpload, UploadDialog) {
                 items = this.$el.find('ol').find('.list-settings-item');
 
             _.each(items, function(element, index) {
-                var key = $(element).find('select').val(),
+                var key = $(element).find('select option:selected').val(),
```

Comment:
To get the value of a select box, you need to use the syntax I added; the previous selection type no longer works.

Think about it as this: the value of a select box isn't actually stored on the select itself. It's the value of the option child that's currently selected.